README.md

```diff
@@ -204,6 +205,12 @@ Unfortunately, it's not possible to get the image of stickers sent by users. You
 
 This feature is not supported on the Messaging API. Users can only play videos that are sent from a bot, they cannot download or forward the videos to other users.
 
+### Why doesn't my audio, video, or image message display properly on Android devices?
+
+The Android LINE App does not currently support Server Name Indication (SNI) on devices that are using versions of Android older than 6.0. As a result, audio, video, and image messages that contain media that is hosted from a server that uses SNI may not be displayed properly on these Android devices. Our team is planning to fix this in a future release.
```

Comment:
@mrmaakun Just a minor change. 

The Android LINE app does not currently support Server Name Indication (SNI) on devices that are using versions of Android older than 6.0. As a result, audio, video, and image messages that are hosted on a server that uses SNI may not be displayed properly on these Android devices. Our team is planning to fix this in a future release.